### PR TITLE
Autonumeric directive now handles disabled state with reactive forms

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Addresses #7

### CODE REVIEW

File: src/app/public/modules/autonumeric/fixtures/autonumeric.component.fixture.html [hunk 1]
```diff
@@ -18,6 +19,7 @@
   <input
     aria-label="Donation amount"
     class="sky-form-control app-template-driven-input"
+    id="donationAmountTemplateDriven"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/autonumeric/fixtures/autonumeric.component.fixture.html]: Is `id` needed for these?

File: src/app/public/modules/autonumeric/autonumeric.directive.ts [hunk 2]
```diff
@@ -72,6 +74,13 @@ export class SkyAutonumericDirective implements OnInit, ControlValueAccessor, Va
     this.updateAutonumericInstance();
   }
 
+  /**
+   * Implemented as part of ControlValueAccessor.
+   */
+  public setDisabledState(value: boolean) {
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/autonumeric/autonumeric.directive.ts]: Return type.

File: src/app/public/modules/autonumeric/fixtures/autonumeric.component.fixture.html (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/autonumeric/fixtures/autonumeric.component.fixture.html]: I added `id` to make getting the elements from the spec files more deliberate.

File: src/app/public/modules/autonumeric/autonumeric.directive.ts (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/autonumeric/autonumeric.directive.ts]: done.

**Pull request merged**